### PR TITLE
Bump 1.11.7

### PR DESCRIPTION
**Pull request by TomSweeneyRedHat**
Bump to v1.11.7 to get ADD url fix for Podman 1.7.0

### CODE REVIEW

**Comment by TomSweeneyRedHat** [comment]
@rhatdan @nalind @vrothberg PTAl
FYI @mheon

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,32 @@
...
+    install.md: update golang dependency
+    imgtype: reset storage opts if driver overridden
+    Start using containers/common
+    overlay.bats typo: fuse-overlays should be fuse-overlayfs
+    chroot: Unmount with MNT_DETACH instead of UnmountMountpoints()
+    bind: don't complain about missing mountpoints
+    imgtype: check earlier for expected manifest type
+    Vendor containers/storage fix
+    Vendor containers/storage v1.15.1
+    Add history names support
```
Comment (vrothberg) [inline CHANGELOG.md]: This would qualify for a minor bump to `v1.12.0`.

Comment (rhatdan) [reply CHANGELOG.md]: I agree, bump the second number,  If we only fix bugs then minor.  If new features like history names then the middle.  If changes in API then major.

Comment (TomSweeneyRedHat) [reply CHANGELOG.md]: Missed that.  Will kill this then.  @rhatdan should I wait for the /var/tmp to land before a new kit and does the history change cause us to NOT want to get this in Podman 1.7.0?

Comment (rhatdan) [reply CHANGELOG.md]: I don't think you should wait for this.

**Comment by TomSweeneyRedHat** [comment]
Will create v1.12.0.  The question will be do it today for Podman v1.7.0 or wait until later.

**Comment by TomSweeneyRedHat** [comment]
OK, I'll go spin up 1.12 then without the /var/tmp fix.

**Pull request closed without merging**